anna/my-app/src/components/movieList/movie-card/MovieCardPrew.tsx

```diff
@@ -0,0 +1,51 @@
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import IMovie from '../movie-card/IMovie';
+import './movie.css';
+
+interface IMovieCardPrewProps {
+  movie: IMovie,
+}
+
+interface IMovieCardPrewState {
+ isError: boolean,
+}
+
+class MovieCardPrew extends React.Component<IMovieCardPrewProps, IMovieCardPrewState> {
+
+  state = {
+    isError: false
+  }
+
+  handleError = () => {
+    this.setState({
+      isError: true
+    })
+    console.log('can`t load image')
+  }
+
+  render() {
+    const { movie } = this.props;
+    const { isError} = this.state;
+
+    return (
+      <Link to={`/film/${movie.id}`} className="movie-link">
+        <div className="movie-card" >
+          {isError && <div className="movie-card-img"></div>}
+          <img src={movie.poster_path} className="movie-card-img" alt="movie poster" onError={this.handleError}
+           style={{
+            display: isError ? "none" : "initial"
+          }}/>
```

Comment:
instead of handling state of image in MovieCard, I would suggest you to write separate simple component that renders image and if it is error, renders some default img.  So, if you needed to render an image in different component, you would just use your component without adding state to a component with an image.